tutorial_R/R_rainclouds.R

```diff
@@ -1,12 +1,20 @@
-### This script creates an R function to generate raincloud plots, then simulates 
-### data for plots. If using for your own data, you only need lines 1-80. 
-### It relies largely on code previously written by David Robinson 
-### (https://gist.github.com/dgrtwo/eb7750e74997891d7c20) and ggplot2 by H Wickham
+### This script creates an R function to generate raincloud plots, then simulates
+### data for plots. If using for your own data, you only need lines 1-80.
+### It relies largely on code previously written by David Robinson
+### (https://gist.github.com/dgrtwo/eb7750e74997891d7c20)
+### and the package ggplot2 by Hadley Wickham
 
-#Load packages
+# Check if required packages are installed ----
+packages <- c("cowplot", "readr", "ggplot2", "dplyr", "lavaan", "smooth", "Hmisc")
+if (length(setdiff(packages, rownames(installed.packages()))) > 0) {
+  install.packages(setdiff(packages, rownames(installed.packages())))
+}
+
+# Load packages ----
```

Comment:
This is so fun that you've helped us out with this! Thank you!

Reply:
:)